File: C++11/백준/Bronze/6763. Speed fines are not fine！/Speed fines are not fine！.cc
#include <iostream>
using namespace std;
int main(){
    int a,b;
    cin >> a >> b;
    if(b<=a){
        cout << "Congratulations, you are within the speed limit!";
    }
    else{
        int cost;
        if(b-a<=20){
            cost = 100;
        }
        else if(b-a<=30){
            cost = 270;
        }
        else{
            cost = 500;
        }
        cout << "You are speeding and your fine is $"<< cost <<'.';
    }
}

File: C#/백준/Bronze/6763. Speed fines are not fine！/Speed fines are not fine！.cs
using System;

namespace q{
    class w{
        static void Main(){
            int a = int.Parse(Console.ReadLine());
            int b = int.Parse(Console.ReadLine());
            if(b<=a){
                Console.WriteLine("Congratulations, you are within the speed limit!");
            }
            else{
                int cost = 0;
                if(b-a<=20){
                    cost = 100;
                }
                else if(b-a<=30){
                    cost = 270;
                }
                else{
                    cost = 500;
                }
                Console.WriteLine($"You are speeding and your fine is ${cost}.");
            }
        }
    }
}